src/test/java/htsjdk/io/HtsPathUnitTest.java

```diff
@@ -169,7 +173,15 @@ public Object[][] invalidHtsPath() {
                 // the nul character is rejected on all of the supported platforms in both local
                 // filenames and URIs, so use it to test HtsPath constructor failure on all platforms
                 {"\0"},
-                {"ftp://broad.org/file with space"} // this has a non-file scheme but isn't encoded properly
+
+                // this has a non-file scheme but isn't encoded properly, best to reject these with
+                // a helpful error message than to continue on and treat it as a file:// path
+                {"ftp://broad.org/file with space"},
+
+                // if you have a scheme you need something with it
+                {"file://"},
```

Comment:
`file://` should be rejected, yes, but do we have a test that proves we accept `file:///` (with 3 slashes, indicating the root directory)?

Reply:
Ah never mind, I see that you do below

Reply:
file:/// should never actually reach this method in the first place, but if it did it should also not be rejected.